### ISSUE TITLE
Failed to clone git repository

### ISSUE THREAD

**Issue by Watnuss** [mssun/passforios#167]
Hi,

I tried several things but always ending up with the same error message, if I try to add my (privately hosted) pass repo.
```
Failed to clone repository from ssh://git@mydomain.net/password-store.git to file:////private/var/mobile/Containers/Shared/AppGroup/<some ID>/Library/password-store-temp
Underlying error: The operation couldn't be completed.
(GTGitErrorDomain error -1.)
```
As I was not able to find anything for this specific error message online, I hope I can get some help here. If you need more information or if I addressed the wrong instance, please let me know.

**Comment by yishilin14** [comment]
Emmm. It is kind of hard to see the reason behind. Not sure whether adding the port in the url would be helpful.

**Comment by nylocx** [comment]
I have very similar symptoms, but I use https as the protocol and I get asked for a password I enter it and nothing happens. I can click clone again and same procedure, after 3 or 4 times there is the preparing repository waiting animation that does not finish (waited 10 minutes).

**Comment by yishilin14** [comment]
V0.5.0 in App Store uses a newer version of ObjevtiveGit framework. Are you still encountering this issue in newer version of Pass?

**Comment by Watnuss** [comment]
Hi, I tried again but the error persists. I get the same error message as before. No change in using the port appended with colon.

**Comment by ivantomica** [comment]
I'm experiencing similar issue as this one. Has anybody found any solution? 

Screenshot of the issue below:

![image](https://user-images.githubusercontent.com/4953199/64646031-75121880-d416-11e9-95e8-50194c4f9568.png)

I'm able to normally clone repository from any other machine without `ssh://` protocol prefix. It is also worth noting that previously I used Pass for iOS without any issues, this issue is encountering on new phone setup.

**Comment by microspino** [comment]
Indeed, happens to me today. As @TrevCan said had to clone the repo again.

**Comment by TrevCan** [comment]
@microspino apparently, it just happened again to me.

I guess the code that runs on this app fails to work through a 'merge'.
I didn't modify the same files on the devices, but what happened is that
I modified the same folder. So maybe the code is returning an error that
says something like 'merge this' and the code itself doesn't implement it
and just assumes that it didn't work at all. 

@mssun I wonder if the 'git' part could be outsourced to another app like, e.g. [Working Copy](https://workingcopyapp.com/). although I must say it is not open source and is paid for *pushing* but this app allows the user to get access to the repo through the Files app. Maybe that's something that other apps could have access to.
